src/styles/main.scss

```diff
@@ -1,3 +1,59 @@
+@import '../styles/variable';
+
 body {
   margin: 0;
+  font-family: $calendar-font;
+  font-size: 30px;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.calendar {
```

Comment:
let's move it to a separate file (according to BEM rules - it's a separate block)